### ISSUE TITLE
Cannot find module after upgrade to NodeJS 12

### ISSUE THREAD

**Issue by alastairpayne** [detrohutt/babel-plugin-import-graphql#60]
After moving from Node 8.11.2 to Node 12.13.0 we started seeing this error for all of our gql files:

```
ERROR in ./src/<...path...>/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: /<...path...>/index.js: Cannot find module 'apollo/queries/user.gql'
(...Stack)
```
Are there any known issues with Node12 and this plugin?

**Comment by GR34SE** [comment]
Have same issue, but strangely importing _graphql-tag_ package on top of the file solves that problem. Looking forward for real solution tho 😄

**Comment by VojtaSim** [comment]
Node 12 changed behaviour of `require.resolve` and thus importing .gql files without absolute or relative path (`./`) will fail.
I've created PR to address this issue https://github.com/detrohutt/babel-plugin-import-graphql/pull/78

**Comment by detrohutt** [comment]
I merged the PR and released v2.8.1.

Closing for now. Reopen if you have any more issues.
